File: Programmers/py/lev2/예상 대진표/solution.py
def solution(n,a,b):
    answer = 0 
    
    while( a != b):
        a,b = (a+1)//2 , (b+1)//2  
        answer+=1
        
    return answer

# 다른 사람 코드 
def solution(n,a,b):
    return ((a-1)^(b-1)).bit_length()

File: Programmers/java/lev2/예상 대진표/solution.java
// 파이썬 코드 참고 
class Solution
{
    public int solution(int n, int a, int b)
    {
        return Integer.toBinaryString((a-1)^(b-1)).length() ;
    }
}

// 내 코드
class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer =0 ;
        while(a != b)
        {
            a = (int)(a+1)/2;
            b = (int)(b+1)/2;
            answer++;
        }
        return answer;
    }
}